src/creators/MarkerClustererCreator.js

```diff
@@ -0,0 +1,127 @@
+import {
+  default as React,
+  PropTypes,
+  Component,
+  Children,
+} from "react";
+import {default as invariant} from "invariant";
+
+import {default as MarkerClustererEventList} from "../eventLists/MarkerClustererEventList";
+import {default as eventHandlerCreator} from "../utils/eventHandlerCreator";
+import {default as defaultPropsCreator} from "../utils/defaultPropsCreator";
+import {default as composeOptions} from "../utils/composeOptions";
+import {default as componentLifecycleDecorator} from '../utils/componentLifecycleDecorator';
+
+import {default as GoogleMapHolder} from "./GoogleMapHolder";
+
+export const markerClustererControlledPropTypes = {
+  // http://google-maps-utility-library-v3.googlecode.com/svn/trunk/markerclustererplus/docs/reference.html
+  averageCenter: PropTypes.bool,
+  calculator: PropTypes.func,
+  clusterClass: PropTypes.string,
+  enableRetinaIcons: PropTypes.bool,
+  gridSize: PropTypes.number,
+  ignoreHidden: PropTypes.bool,
+  imageExtension: PropTypes.string,
+  imagePath: PropTypes.string,
+  imageSizes: PropTypes.array,
+  maxZoom: PropTypes.number,
+  minimumClusterSize: PropTypes.number,
+  styles: PropTypes.array,
+  title: PropTypes.string,
+  zoomOnClick: PropTypes.bool,
+};
+
+export const markerClustererDefaultPropTypes = defaultPropsCreator
+  (markerClustererControlledPropTypes);
+
+const markerClustererUpdaters = {
+  averageCenter       (averageCenter, component) { component.getMarkerClusterer().setAverageCenter(averageCenter); },
+  calculator          (calculator, component) { component.getMarkerClusterer().setCalculator(calculator); },
+  clusterClass        (clusterClass, component) { component.getMarkerClusterer().setClusterClass(clusterClass); },
+  enableRetinaIcons   (enableRetinaIcons, component) { component.getMarkerClusterer().setEnableRetinaIcons(enableRetinaIcons); },
+  gridSize            (gridSize, component) { component.getMarkerClusterer().setGridSize(gridSize); },
+  ignoreHidden        (ignoreHidden, component) { component.getMarkerClusterer().setIgnoreHidden(ignoreHidden); },
+  imageExtension      (imageExtension, component) { component.getMarkerClusterer().setImageExtension(imageExtension); },
+  imagePath           (imagePath, component) { component.getMarkerClusterer().setImagePath(imagePath); },
+  maxZoom             (maxZoom, component) { component.getMarkerClusterer().setMaxZoom(maxZoom); },
+  minimumClusterSize  (minimumClusterSize, component) { component.getMarkerClusterer().setMinimumClusterSize(minimumClusterSize); },
+  styles              (styles, component) { component.getMarkerClusterer().setStyles(styles); },
+  title               (title, component) { component.getMarkerClusterer().setTitle(title); },
+  zoomOnClick         (zoomOnClick, component) { component.getMarkerClusterer().setZoomOnClick(zoomOnClick); },
```

Comment:
Every official google maps instance provide a public API call `setOptions`: http://git.io/vCWHh. Does marker clusterer have the same API? If yes, please add it as others. If no, just leave it as is now. :+1: